packages/lit-html/src/lit-html.ts

```diff
@@ -518,7 +518,9 @@ const getTemplateHtml = (
           end
         : s +
           marker +
-          (attrNameEndIndex === -2 ? (attrNames.push(undefined), i) : end);
+          (attrNameEndIndex === -2
+            ? (attrNames.push(undefined), String(i))
```

Comment:
I'd rather not do this. The `+` operator will already do the coercion, so this is just extra bytes. How about a cast?